### PR TITLE
Create PR template

### PR DESCRIPTION
**Pull request by TheWeaVer**
Create PR template

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+**Motivation**:
```
Comment (nullist0) [inline .github/PULL_REQUEST_TEMPLATE.md:1]: I think that this is better to use header, instead of bold.

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+
+Explain why you're making this change and what problem you're trying to solve.
```
Comment (nullist0) [inline .github/PULL_REQUEST_TEMPLATE.md:3]: The guide text is better to show as a comment. `<!-- -->`. The deletion will occur too many times to do that.

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 1 above)
Comment (TheWeaVer) [reply .github/PULL_REQUEST_TEMPLATE.md:1]: Which header should you recommend?

Comment (nullist0) [reply .github/PULL_REQUEST_TEMPLATE.md:1]: I don't care about the header level, but I'm using the second header `##` in the company.

Comment (TheWeaVer) [reply .github/PULL_REQUEST_TEMPLATE.md]: I use `###` 😄

**Review by nullist0 — APPROVED** [review]
LGTM

**Pull request merged**